apostrophe-2/inc/jetpack.php

```diff
@@ -61,13 +61,23 @@ function apostrophe_2_has_post_thumbnail( $post = null ) {
 }
 
 /**
- * Return early if Social Menu is not available.
+ * Return early if Social Menu is not available and check if it should be displayed on mobile.
  */
 function apostrophe_2_social_menu() {
 	if ( ! function_exists( 'jetpack_social_menu' ) ) {
 		return;
-	} else {
+	} 
+	
+	if ( function_exists( 'jetpack_social_menu' ) && ( get_theme_mod( 'apostrophe_2_mobile_social' ) == 0 )) {
```

Comment:
I think this logic can be simplified; we already know the `jetpack_social_menu` function exists since we checked for it early on, so those can be removed:

```
        if ( get_theme_mod( 'apostrophe_2_mobile_social' ) == 0 ) {
		jetpack_social_menu();
	} else { ?>
		<div class="social-mobile-menu">
			<?php jetpack_social_menu(); ?>
		</div>
	<?php }
```

Reply:
Hi @sixhours, I updated this PR to remove that markup and favour the body tag approach like you suggested, along with moving the menu to Content Options. Let me know what you think. :)